src/iconsManifest/supportedExtensions.ts

```diff
@@ -3961,6 +3961,7 @@ export const extensions: IFileCollection = {
     { icon: 'vala', extensions: ['vala'], format: FileFormat.svg },
     { icon: 'vapi', extensions: ['vapi'], format: FileFormat.svg },
     { icon: 'vash', extensions: ['vash'], light: true, format: FileFormat.svg },
+    { icon: 'vapor', extensions: ['vapor.yml'], format: FileFormat.svg },
```

Comment:
Is this intended to be an extension like `a.vapor.yaml`? Or will it be the name of a file? If it's the later, then you should add `filename: true` and that will make it (similar to [this](https://github.com/vscode-icons/vscode-icons/blob/cf315a35e926fb9f225e15406de8316f41ad7c7e/src/iconsManifest/supportedExtensions.ts#L2662-L2672))

Reply:
Ok, thanks for the suggestions, changing asap